contracts/MarginCalculator.sol

```diff
@@ -112,18 +119,24 @@ contract MarginCalculator is Ownable {
         bool _isPut,
         uint256 _timeToExpiry
     ) external onlyOwner {
+        // get product hash
         bytes32 productHash = _getProductHash(_underlying, _strike, _collateral, _isPut);
+        // get array of expiries
         uint256[] storage expiryArray = productTimeToExpiry[productHash];
 
+        // check that this is the first expiry to set, if not the last expiry should be less than the new one to insert (to make sure the array stay in order)
         require(
             (expiryArray.length == 0) || (_timeToExpiry > expiryArray[expiryArray.length.sub(1)]),
             "MarginCalculator: expiry array is not in order"
         );
+
+        // check that this time to expiry have an upper bound value (setTimeToExpiryValue() should be called before this)
```

Comment:
can we ask Andrew to rename this variable? `timeToExpiryValue` is causing confusion as it is so meaningless

Reply:
Open a thread about that, can change it later in another PR.